### PR TITLE
[d3d9] Mark mips as dirty on Clear if needed

### PR DESCRIPTION
**Pull request by DadSchoorse**
Fixes a Wine test.

### CODE REVIEW

File: src/d3d9/d3d9_device.cpp [hunk 1]
```diff
@@ -1478,8 +1478,14 @@ namespace dxvk {
         for (auto rt : m_state.renderTargets) {
           auto rtv = rt != nullptr ? rt->GetRenderTargetView(srgb) : nullptr;
 
-          if (unlikely(rtv != nullptr))
+          if (unlikely(rtv != nullptr)) {
             ClearImageView(fullClear, offset, extent, rtv, VK_IMAGE_ASPECT_COLOR_BIT, clearValueColor);
+
+            D3D9CommonTexture* dstTexture = rt->GetCommonTexture();
```
Comment (Joshua-Ashton) [inline src/d3d9/d3d9_device.cpp:1484]: Just do rt->GetCommonTexture()->IsAutomaticMip()

**Review by Joshua-Ashton — CHANGES_REQUESTED** [review]

File: src/d3d9/d3d9_device.cpp (hunk 1 above)
Comment (DadSchoorse) [reply src/d3d9/d3d9_device.cpp:1484]: Then I need to call rt->GetCommonTexture() twice, MarkTextureMipsDirty needs it as well.

Comment (Joshua-Ashton) [reply src/d3d9/d3d9_device.cpp:1484]: derp...

**Comment by Joshua-Ashton** [comment]
Needs rebase

**Pull request merged**